### PR TITLE
Add ukrainian language to the PIN code flow

### PR DESCRIPTION
**Pull request by Pavlo-Hural**
Added ukrainian translation

### CODE REVIEW

File: app/src/main/res/values-uk/strings_pin.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pin">Пароль</string>
```
Comment (yterletskyi) [inline app/src/main/res/values-uk/strings_pin.xml]: can you please check if we really need to have this string (for `en` too )? It seems to me that we use it somewhere and then it's overriden by something.

File: app/src/main/res/values-uk/strings_pin.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    <string name="setupPin">Налаштування паролю</string>
```
Comment (yterletskyi) [inline app/src/main/res/values-uk/strings_pin.xml]: can you please change it to be `setup_pin` to keep naming convention

File: app/src/main/res/values-uk/strings_pin.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <string name="phrase1">Давайте створимо PIN-код, щоб \n захистити ваші ідеї.</string>
+    <string name="phrase2">Ми запитуватимемо його кожного разу, коли ви \n відкриватимете програму.</string>
+    <string name="phrase3">Будьте уважні, відновити \n втрачений ПІН наразі неможливо.</string>
+    <string name="action_button_next">Далі</string>
+    <string name="pin_enter_title">Введіть пароль</string>
+    <string name="pin_repeat_title">Повторіть пароль</string>
+    <string name="pin_not_match_error">Пароль не співпадає</string>
```
Comment (yterletskyi) [inline app/src/main/res/values-uk/strings_pin.xml]: Краще `Паролі не співпадають`

File: app/src/main/res/values-uk/strings_pin.xml (hunk 1 above)
Comment (Pavlo-Hural) [reply app/src/main/res/values-uk/strings_pin.xml]: We used this string in mobile_navigation in label 
image

Comment (yterletskyi) [reply app/src/main/res/values-uk/strings_pin.xml]: right, but is the label visible somewhere?

Comment (Pavlo-Hural) [reply app/src/main/res/values-uk/strings_pin.xml]: i guess this label is not visible, because we haven't ActionBar in pin fragment, right?

**Review by yterletskyi — APPROVED** [review]

**Pull request merged**